README.md

```diff
@@ -22,19 +20,41 @@ Usage:
 Available Commands:
   dinghy      Run dinghy subcommands
   help        Help about any command
+  version     Prints version information
 
 Flags:
-  -h, --help   help for arm
+  -h, --help              help for arm
+  -l, --loglevel string   log level (default "info")
 
 Use "arm [command] --help" for more information about a command.
 ```
 
+##Mac OS
+Since arm-cli is not signed you may receive a couple of messages regarding security. To execute the binaries you need to:
+
+1.- When you execute arm-cli on the console you may see this message.
```

Comment:
I think the dashes (`-`) break rendering too :( Any reason to keep them?

Reply:
Changed them also!! Thanks!